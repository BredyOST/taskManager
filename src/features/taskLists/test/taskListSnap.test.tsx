import {render} from "@testing-library/react";
import TaskLists from "../taskLists.tsx";


describe('TaskListSnap', () => {
    test('matches snapshot', () => {
        const {asFragment} = render(<TaskLists openList={true}/>)
        expect(asFragment()).toMatchSnapshot()
    })
})