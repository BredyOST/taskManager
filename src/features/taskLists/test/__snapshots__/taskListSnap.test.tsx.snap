// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TaskListSnap matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="sc-blHHSb dmbdEB"
  />
</DocumentFragment>
`;
